action-server/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "covidflow"
+version = "0.1.0"
+description = ""
+authors = [""]
+
+[tool.poetry.dependencies]
+pydantic = "^1.4"
+python = "^3.6"
+rasa-sdk = "^1.9.0"
```

Comment:
Is it problematic to "reinstall" the rasa-sdk in the rasa image?

Reply:
Doesn't seem to be a problem